test/cli/verbose.test.js

```diff
@@ -33,4 +37,37 @@ describe('newman run --verbose', function () {
             done();
         });
     });
+
+    it('should limit to 2KB with --verbose option only without setting output-size', function (done) {
```

Comment:
Very neat and creative test. Like it ❤️😊. 

Another improvement could be if the body was autogenerated series of number marking increasing size. That way, we can both assert points and accuracy.